website/app/index.html

```diff
@@ -13,6 +13,10 @@
     <link rel="shortcut icon" href="/assets/logos/favicon.png" />
 
     {{content-for "head-footer"}}
+
+    <!-- Fathom - beautiful, simple website analytics -->
```

Comment:
@Dhaulagiri do you think we can use an approach like this:
https://github.com/hashicorp/design-system/blob/main/packages/ember-flight-icons/index.js
in which we have a custom `content-for` and conditionally we check, where we define it, if to add the code snippet or not, depending on the ENV variable (that should know if we are in production or not)

Reply:
good idea. let me take a look at that. there's supposed to be a built in way for fathom to do this but a) our domain doesn't exist yet and b) even when it does it feels harder than it should be

Reply:
I did refactor to only inject this if a `FATHOM_ENABLED` environment variable is set, which we could set _only_ on the main production build. Let me know what you think.

Reply:
Do we want to drop a line in the readme or somewhere to document this variable? `ember-cli-dotenv` seem to be throwing a warning if not set.

Reply:
Yes, I have an open followup to add some documentation.

I'd also like to see if this warning can not happen since it is not useful

Reply:
🙌